### PR TITLE
refactor(LSC): replace deprecated functions under pkg io/ioutil

### PR DESCRIPTION
**Pull request by xinqshang**
## Pre-Checklist

Note: please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](https://github.com/devstream-io/devstream/blob/main/CONTRIBUTING.md) documentation.
- [ ] My code has the necessary comments and documentation (if needed).
- [ ] I have added relevant tests

## Description
  golangci-lint report some err when exec make all
  to fix it
  replace deprecated functions under pkg io/ioutil

## Related Issues
N/A

## New Behavior (screenshots if needed)
N/A

### CODE REVIEW

**Comment by daniel-hutao** [comment]
I'm so sorry that this branch has conflicts for my previous merged refactor pr. Please resolve the conflicts and then, we'll take a review right away.

**Comment by aFlyBird0** [comment]
Hi @xinqshang , thanks for your contribution! Please [join CNCF's Slack](https://slack.cncf.io), and join our [channel in CNCF Slack](https://cloud-native.slack.com/messages/devstream)!

Here are a few useful things for you to continue your open-source journey:

- If you are searching for something else to work on, please see ["find an issue"](https://docs.devstream.io/en/latest/contributing_guide/#find-an-issue).
- Check out our [new contributor guide](https://docs.devstream.io/en/latest/contributing_guide/#new-contributor-guide) and the ["Developer Guide"](https://docs.devstream.io/en/latest/development/development-workflow/).
- Read our [blog](https://blog.devstream.io/); they are on [Medium](https://medium.com/devstream), too.

We will review your pr tomorrow morning.🙌

**Comment by iyear** [comment]
If you find it difficult to resolve a large number of conflicts and the code submitted is not too difficult to modify, you can close the PR first, sync fork and then submit a new PR again.

**Comment by xinqshang** [comment]
> I'm so sorry that this branch has conflicts for my previous merged refactor pr. Please resolve the conflicts and then, we'll take a review right away.

done

**Comment by xinqshang** [comment]
> Hi @xinqshang , thanks for your contribution! Please [join CNCF's Slack](https://slack.cncf.io), and join our [channel in CNCF Slack](https://cloud-native.slack.com/messages/devstream)!
> 
> Here are a few useful things for you to continue your open-source journey:
> 
> * If you are searching for something else to work on, please see ["find an issue"](https://docs.devstream.io/en/latest/contributing_guide/#find-an-issue).
> * Check out our [new contributor guide](https://docs.devstream.io/en/latest/contributing_guide/#new-contributor-guide) and the ["Developer Guide"](https://docs.devstream.io/en/latest/development/development-workflow/).
> * Read our [blog](https://blog.devstream.io/); they are on [Medium](https://medium.com/devstream), too.
> 
> We will review your pr tomorrow morning.🙌

got it

**Review by iyear — APPROVED** [review]
Careful Gopher🐭!

File: pkg/util/k8s/static.go [hunk 1]
```diff
@@ -31,10 +31,10 @@ func (c *Client) ListDevstreamNamespace() (*corev1.NamespaceList, error) {
 
 // Check whether the given namespace is created by dtm
 // If the given namespace has label "created_by=DevStream", we'll control it.
-// 1. The specified namespace is created by dtm, then it should be deleted
-//    when errors are encountered during creation or `dtm delete`.
-// 2. The specified namespace is controlled by user, maybe they want to deploy plugins in
-//    an existing namespace or other situations, then we should not delete this namespace.
+//  1. The specified namespace is created by dtm, then it should be deleted
```
Comment (daniel-hutao) [inline pkg/util/k8s/static.go:34]: Unexpected space here.

Comment (daniel-hutao) [reply pkg/util/k8s/static.go:34]: I'll help to fix it and merge this pr asap.

**Review by daniel-hutao — APPROVED** [review]

**Pull request merged**